test/gtest/tensor_api.cpp

```diff
@@ -47,7 +47,7 @@
 
 // miopenLastDataType must be changed if new data types are added
 #define miopenFirstDataType miopenHalf
-#define miopenLastDataType miopenBFloat8
+#define miopenLastDataType miopenInt64
```

Comment:
[R] https://github.com/ROCm/MIOpen/labels/value_low Introduce `miopenDataTypeLast_` in miopen.h right in this PR (and I do not see any need to guard it with BETA macro).

Reply:
`miopenInt64` is the last type in both cases (beta/not-beta).
Now it is even safer, since it does not require `MIOPEN_BETA` macro.

Actually I'm a bit against of `first-last` values, especially how it is used in the test.
For the common case or iteration over the enum it may be helpful, but definitely not for this test.